client/src/index.tsx

```diff
@@ -9,6 +9,12 @@ import "./index.css";
 import App from "./components/app";
 import store from "./reducers";
 
+import "./assets/img/safari.png";
```

Comment:
can we add a comment to indicate that these are imported for Webpack `asset/resource` module to bundle in order to be used in obsolete browser template 🙏 ?

Thank you!